roles/disaster_recovery/README.md

```diff
@@ -0,0 +1,93 @@
+| dr_reset_mac_pool       | True                  | If True, then once a VM will be registered, it will automatically reset the mac pool, if configured in the VM.        |
+| dr_cleanup_retries_maintenance       | 3                  | Specify the number of retries of moving a storage domain to maintenance VM as part of a fail back scenario.       |
+| dr_cleanup_delay_maintenance       | 120                  | Specify the number of seconds between each retry as part of a fail back scenario.       |
+| dr_clean_orphaned_vms        | True                  | Specify whether to remove any VMs which have no disks from the setup as part of cleanup.       |
+| dr_clean_orphaned_disks        | True                  | Specify whether to remove lun disks from the setup as part of engine setup.       |
+| dr_running_vms		 | /tmp/ovirt_dr_running_vm_list	 | Specify the file path which is used to contain the data of the running VMs in the secondary setup before the failback process run on the primary setup after the secondary site cleanup was finished. Note that the /tmp folder is being used as default so the file will not be available after system reboot.
+
+
+
+Dependencies
+------------
+
+No.
+
+Example Playbook
+----------------
+
+```yaml
+---
+- name: Setup oVirt environment
+  hosts: localhost
+  connection: local
+  vars_files:
+     - ovirt_passwords.yml
+     - disaster_recovery_vars.yml
+  roles:
+     - {role: oVirt.disaster-recovery}
+```
+
+Generate var file mapping [demo](https://youtu.be/s1-Hq_Mk1w8)
+<br/>
+Fail over scenario [demo](https://youtu.be/mEOgH-Tk09c)
+
+Scripts
+-------
+The ovirt-dr script should provide the user a more convenient way to run
+disaster recovery actions as a way to avoid using ansible playbooks directly.
+There are four actions which the user can execute:
+- `generate`	Generate the mapping var file based on the primary and secondary setup, to be used for failover and failback
+- `validate`	Validate the var file mapping which is used for failover and failback
+- `failover`	Start a failover process to the target setup
+- `failback`	Start a failback process from the target setup to the source setup
+
+Each of those actions are using a configuration file whose default location is `oVirt.disaster-recovery/files/dr.conf`<br/>
+The configuration file's location can be changed using `--conf-file` flag in the `ovirt-dr` script.<br/>
+Log file and log level can be configured as well through the `ovirt-dr` script using the flags `--log-file` and `--log-level`
+
+
+Example Script
+--------------
+For mapping file generation:
+```console
+$ ./ovirt-dr generate --log-file=ovirt-dr.log --log-level=DEBUG
+```
+For mapping file validation:
+```console
+$ ./ovirt-dr validate
+```
+For fail-over operation:
+```console
+$ ./ovirt-dr failover
+```
+For fail-back operation:
+```console
+$ ./ovirt-dr failback
+```
+
+License
+-------
+
+Apache License 2.0
+
```

Comment:
Remove License

Reply:
Done, removed.

Reply:
> Remove License

BTW: I see that "hosted_engine_setup" still has the licence line at the end of the readme file. So should this line be removed?